modules/button/react/lib/ButtonStyles.ts

```diff
@@ -456,7 +456,7 @@ function getButtonFocusRing(buttonType: AllButtonTypes): CSSObject {
     case TextButtonTypes.Inverse:
     case TextButtonTypes.AllCaps:
     case TextButtonTypes.InverseAllCaps:
-      return {};
+      return focusRing(2, 2);
```

Comment:
Changing this to `focusRing(2, 2)`, otherwise the inverse will not get the white border. Is that ok?

Reply:
This makes the `Default` and `AllCaps` versions' focus ring 2px too large. Those two types should be moved up to line 452. We needed to delete the `focusRingInner` and `focusRingOuter` in `Default`, but not in `Inverse`. Looks like it's disappeared in both. 

While experimenting with it locally I ended up just making the change so I'll just push my commit 🙂